docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -472,7 +472,9 @@ query brands {
 
 ## Complexity limits
 
-The GraphQL Storefront API uses an algorithm to calculate a complexity score for queries made against the API. Queries that exceed the complexity score will receive an error response:
+The GraphQL Storefront API uses an algorithm to calculate a complexity score for queries made against the API. The API returns the query complexity as an integer in the HTTP response header `x-bc-graphql-complexity` when you send a valid GraphQL request.
+
+Queries whose complexity score exceed the limit will receive an error response:

```

Comment:
@bc-andreadao I would change this sentence to say the following: "If the complexity score of a query exceeds the complexity limit, the following error code will be sent:"

Also, can the complexity limit be updated (like from 1000 to 2000 for example?)

Reply:
@stanzikratelbc I copyedited to change the sentence structure and fix the grammar, but made it active voice

Thanks for bringing up the update for the limit. The attached screenshot that I just received from @6juara9 has a complexity limit of 10000, so I updated the doc.
<img width="1680" alt="complexity limit" src="https://user-images.githubusercontent.com/96258747/211917716-51432856-3c13-4125-aee6-12d3f921c37d.png">